src/settings/LoanPolicy/components/EditSections/LoansSection/LoansSection.js

```diff
@@ -150,14 +153,28 @@ class LoansSection extends React.Component {
           </div>
         }
         { policy.isLoanable() &&
-          <div data-test-loans-section-grace-period>
-            <Period
-              fieldLabel="ui-circulation.settings.loanPolicy.gracePeriod"
-              inputValuePath="loansPolicy.gracePeriod.duration"
-              selectValuePath="loansPolicy.gracePeriod.intervalId"
-              intervalPeriods={this.generateOptions(intervalPeriods, 'ui-circulation.settings.loanPolicy.selectInterval')}
-              changeFormValue={change}
-            />
+          <div>
```

Comment:
```suggestion
          <Fragment>
```
Will fragment help to avoid wrapper div? Make sure to import Fragment from React if it will.